src/main/app/case/to-api-format.ts

```diff
@@ -316,13 +316,13 @@ const fields: ToApiConverters = {
   coApplicant2StatementOfTruth: data => ({
     coApplicant2StatementOfTruth: checkboxConverter(data.coApplicant2StatementOfTruth),
   }),
-  applicant1WhoIsFinancialOrderFor: data => ({
-    applicant1FinancialOrdersFor:
-      data.applicant1ApplyForFinancialOrder === YesOrNo.YES ? data.applicant1WhoIsFinancialOrderFor : [],
+  applicant1ApplyForFinancialOrder: data => ({
```

Comment:
Could you have a comment on this explaining the change maybe?

Reply:
In the previous user journey, we had 1 form that contained radio buttons to select whether they wanted a financial order (`applicant1ApplyForFinancialOrder`) and checkboxes to select who it should apply to (`applicant1WhoIsFinancialOrderFor`). They submitted the form which contained data for both fields and if they had selected No for `data.applicant1ApplyForFinancialOrder`, we discarded any checkbox selections for `applicant1WhoIsFinancialOrderFor` by it that to a blank array.

In the new user journey, `applicant1ApplyForFinancialOrder` and `applicant1WhoIsFinancialOrderFor` are answered on different pages in different forms. We never receive the form data with both answers, but if at any point they answer No to `applicant1ApplyForFinancialOrder`, we add `applicant1FinancialOrdersFor: []` to the body of the post request to ensure we blank out that field in their case data. For example, they could begin their application initially by selection Yes for `applicant1ApplyForFinancialOrder` and choosing an option for `applicant1WhoIsFinancialOrderFor`. Then if they then go back and change `applicant1ApplyForFinancialOrder` to No, we want to ensure that `applicant1WhoIsFinancialOrderFor` is not still set in the database.